tests/baselines/reference/arrayFakeFlatNoCrashInferenceDeclarations.types

```diff
@@ -7,7 +7,7 @@ type BadFlatArray<Arr, Depth extends number> = {obj: {

```

Comment:
Uh, is this change ok @weswigham?

Reply:
I'm... Not even sure what's different at a glance

Reply:
<img width="1440" alt="Screen Shot 2021-07-06 at 3 44 07 PM" src="https://user-images.githubusercontent.com/3277153/124670433-38c46d80-de71-11eb-9d79-1f2862c07315.png">

Reply:
Uhh... Yeah, it's probably fine - there's an `any` due to truncation in the removed type text, so all we did was truncate more, which is generally fine.